src/app/p/[slug]/page.tsx

```diff
@@ -92,6 +93,18 @@ export default async function PublicPage({
             Tanya ke {owner?.data?.name}
           </h1>
 
+          {owner.data?.x_username && (
```

Comment:
The owner can be null somehow, how about adding optional chain also?


```suggestion
          {owner?.data?.x_username && (
```